### ISSUE TITLE
angular_components has no version 0.7.1

### ISSUE THREAD

**Issue by alexd1971** [dart-lang/angular_components#167]
Documentation describes installation:
```
dependencies:
  angular_components: "^0.7.1"
```
I get following:
```
$ pub get
Resolving dependencies...
Package angular_components has no versions that match >=0.7.1 <0.8.0 derived from:
- project depends on version ^0.7.1
exit code 1
```

**Comment by nshahan** [comment]
There is a 0.7.1 version published. You must have a dependency on an older version of another package that is preventing the version solver from getting it. Try running `pub upgrade` to update all of your other dependencies. If that fails try running `pub deps` to see what versions you are getting. Maybe we can find the package that is preventing the upgrade.

**Comment by 4cm4k1** [comment]
@alexd1971 For what it's worth, I was getting this when I was trying to install the 5.0.0-alpha of [Angular](https://github.com/dart-lang/angular). I went back to 4.0.0 and no issue. (BUT, I know v5 of Angular only was released in the last few hours…)

**Comment by nshahan** [comment]
An updated version has been published to support angular: 5.0.0-alpha.
https://pub.dartlang.org/packages/angular_components/versions/0.9.0-alpha

Closing issue, feel free to reopen if you are still having problems finding a compatible version.
